core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/YouTubePlayerView.kt

```diff
@@ -90,11 +90,29 @@ class YouTubePlayerView(
       legacyTubePlayerView.initialize(
         youTubePlayerListener,
         handleNetworkEvents,
-        IFramePlayerOptions.default
+        IFramePlayerOptions.default,
+        videoId
       )
     }
   }
 
+  /**
+   * Initialize the player. You must call this method before using the player.
+   * @param youTubePlayerListener listener for player events
+   * @param handleNetworkEvents if set to true a broadcast receiver will be registered and network events will be handled automatically.
+   * If set to false, you should handle network events with your own broadcast receiver.
+   * @param playerOptions customizable options for the embedded video player.
+   * @param videoId optionally used to load an initial video.
+   */
+  fun initialize(youTubePlayerListener: YouTubePlayerListener, handleNetworkEvents: Boolean, playerOptions: IFramePlayerOptions, videoId: String?) {
```

Comment:
Instead of adding a new method, let's reuse the existing one. You can change the signature to

```
fun initialize(youTubePlayerListener: YouTubePlayerListener, handleNetworkEvents: Boolean, playerOptions: IFramePlayerOptions, videoId: String? = null)
```